package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "lyuma.av3emulator",
```

Comment:
In my opinion, It's much better to use `io.github.lyuma` as prefix of the package. 
lyuma.github.io is domain that usable by github user lyuma (as github pages) and `io.github.<github user>` is used as prefix for who doesn't own any domain in mavem central, another package system that uses reverse domain notation.

Reply:
Im fine with using any prefix, but this still has Lyumas name in the package, so I don't know how they feel about it @lyuma